### PR TITLE
Fix some map tests and added them to testSuite.xml

### PR DESCRIPTION
**Pull request by RodriGomez**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/interactivemaps/InteractiveMapPageObject.java [hunk 1]
```diff
@@ -276,6 +276,13 @@ public void verifyMapOpened() {
     PageObjectLogging.log("verifyMapOpened", "Map was opened", true);
   }
 
+  public void verifyMapOpenedForDeleteMapTests() {
+    waitForElementVisibleByElement(mapFrame);
+    driver.switchTo().frame(mapFrame);
+    driver.switchTo().defaultContent();
+    PageObjectLogging.log("verifyMapOpenedForDeleteMapTests", "Map was opened", true);
+  }
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/interactivemaps/InteractiveMapPageObject.java]: Why switching? Can't you just see if element is present?

Comment (RodriGomez) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/interactivemaps/InteractiveMapPageObject.java]: The map is inside an iframe and it is not possible to view elements within it without switching.

Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/interactivemaps/InteractiveMapPageObject.java]: But, you'r not searching for any element after 
```
driver.switchTo().frame(mapFrame);
```
you could just look for iFrame element,w ithout switching

Comment (RodriGomez) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/interactivemaps/InteractiveMapPageObject.java]: Fixed.

**Comment by ludwikkazmierczak** [comment]
LGTM

**Pull request merged**